src/CacheInterface.php

```diff
@@ -2,12 +2,22 @@
 
 namespace React\Cache;
 
+use React\Promise\PromiseInterface;
+
 interface CacheInterface
 {
-    // @return React\Promise\PromiseInterface
+    /**
+     * @return PromiseInterface
+     */
     public function get($key);
 
+    /**
+     * @return PromiseInterface
+     */
     public function set($key, $value);
 
+    /**
+     * @return PromiseInterface
```

Comment:
This is debatable, but psr/simple-cache as well as doctrine/cache return true/false for set and delete/remove.

https://github.com/php-fig/simple-cache/blob/master/src/CacheInterface.php#L29
https://github.com/php-fig/simple-cache/blob/master/src/CacheInterface.php#L41

https://github.com/doctrine/cache/blob/master/lib/Doctrine/Common/Cache/Cache.php#L77
https://github.com/doctrine/cache/blob/master/lib/Doctrine/Common/Cache/Cache.php#L86